### PR TITLE
Removes stability material property from the game

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
All uses of stability (used only for custom small cells and custom walls) and the related property have been removed.

## Why's this needed? 
Previously, the stability property served to encourage null alloying low stability materials with high stability materials. With the removal of null alloying, stability no longer has much of a point outside of rng cell exploding. This also helps to trim up matsci in order to help with further expansion.

## Changelog

```changelog
(u)Ikea
(+)The stability material property has been removed from the game.
```

### CODE REVIEW

File: code/turf/walls.dm [hunk 1]
```diff
@@ -342,18 +331,6 @@
 		else return
 
 	else
-		if(src.material)
-			src.material.triggerOnHit(src, W, user, 1)
```
Comment (Tarmunora) [inline code/turf/walls.dm:346]: why remove the matProc trigger?

File: code/turf/walls.dm [hunk 2]
```diff
@@ -536,18 +513,6 @@
 	if(istype(src, /turf/simulated/wall/r_wall) && src.d_state > 0)
 		src.icon_state = "r_wall-[d_state]"
 
-	if(src.material)
-		src.material.triggerOnHit(src, W, user, 1)
```
Comment (Tarmunora) [inline code/turf/walls.dm:540]: ditto here

**Review by Tarmunora — CHANGES_REQUESTED** [review]
also you missed one in `code/turf/turf_autoalign.dm`

File: code/turf/walls.dm (hunk 1 above)
Comment (ThatFiveGuys) [reply code/turf/walls.dm:346]: Oops didnt really think that one through Ill readd it

Comment (ThatFiveGuys) [reply code/turf/walls.dm]: Done! Ditto for other

**Pull request merged**